lib/matplotlib/tests/test_style.py

```diff
@@ -11,7 +11,7 @@
 import pytest
 
 import matplotlib as mpl
-from matplotlib import style
+from matplotlib import pyplot as plt, style
```

Comment:
Just out of curiosity, is this import style "ok"?

Reply:
Yeah, doing `import sys, os` is what is discouraged.  

In that case that you are importing mulitple top level things is obscured, in this case we are importing multiple things from the same namespace (and renaming one).  https://www.python.org/dev/peps/pep-0008/#imports